x-pack/legacy/plugins/file_upload/public/legacy.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { npStart } from 'ui/new_platform';
+import { plugin } from '.';
+
+const pluginInstance = plugin();
```

Comment:
Why do we need this module hopping? Is this something from the platform?

```
import { FileUploadPlugin } from './plugin';

const pluginInstance = new FileUploadPlugin();
```

Reply:
Again just sticking with the [expected shape](https://github.com/elastic/kibana/blob/master/src/core/MIGRATION.md#3-export-your-runtime-contract) for consistency with other plugins at this stage. Ultimately this file will go away when moved though.